nnpdf_data/theory.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class _TheoryCard:
+    ID: int
+    PTO: int
+    FNS: str
+    DAMP: int
+    DAMPPOWERc: None #Power of the damping factor in FONLL for the c
+    DAMPPOWERb: None #Power of the damping factor in FONLL for the b
+    IC: int
+    IB: int
+    ModEv: str
+    ModSV: None#
+    XIR: float
+    XIF: float
+    NfFF: int
+    #MaxNfAs: int
+    #MaxNfPdf: int
+    nfref: 5
+    nf0: 3
+    Q0: float
+    alphas: float
+    Qref: float #TODO: understand if we want to enforce Qref = Qedref and remove Qedref. Note that this is mandatory in eko
+    QED: int
+    alphaqed: float
+    Qedref: float
+    SxRes: int#
+    SxOrd: str#
+    HQ: str
+    mc: float
+    Qmc: float#
```

Comment:
Can you remind me where `Qmc`, `Qmb` and `Qmt` are used? @felixhekhorn

Reply:
In eko [here](https://github.com/NNPDF/eko/blob/be97472bfe199fbf105d19d4b092aaa757488852/src/eko/io/runcards.py#L219), these are the reference scales for the MSbar masses